source/user-manual/api/rbac/configuration.rst

```diff
@@ -113,15 +118,15 @@ The link between users and policies is **roles**. Therefore, for the previous ex
     }
 
 .. note::
-    The highlighted lines are designed for a future feature, still in development. It does not affect the functionality.
+    The highlighted lines are intended for future feature still under development. They currently have no effect.
 
-The request with the information showed above would look like this:
+As before, the creation of that role can be requested using an API endpoint. In this case, the request for the role shown above would look like this:
 
 .. code-block:: console
 
     # curl -X POST "https://localhost:55000/v4/security/roles?pretty=true" -H  "Authorization: Bearer <YOUR_JWT_TOKEN>" -d "{\"name\":\"sales-team\",\"rule\":{\"MATCH\":{\"definition\":\"sales-team\"}}}"
 
-The response body would be this. Remember that the ID is needed to link policies to this role.
+The response body would be similar to this one. It is important to remember the ID as it will be needed to link policies to this role.
```

Comment:
```suggestion
The response body would be similar to this one. It is important to remember the role ID as it will be needed to link policies to this role.
```